pkg/schedule/handler/handler.go

```diff
@@ -808,7 +808,7 @@ func (h *Handler) GetSchedulerByStatus(status string, needTS bool) (interface{},
 	switch status {
 	case "paused":
 		pausedSchedulers := make([]string, 0, len(schedulers))
-		pausedPeriods := []schedulerPausedPeriod{}
```

Comment:
it will return nil rather than `[]` when it is empty, which will make some qa tests failed.

Reply:
get it. fixed.